packages/fints/typings/pain.001.001.03.d.ts

```diff
@@ -0,0 +1,390 @@
+import * as Primitive from './xml-primitives';
```

Comment:
I guess this wile and the other `.d.ts` file were autogenerated. Instead of checking in autogenerated code, please add a yarn command for generating them, perhaps into a directory `generated` in the package's root directory and add it to the Makefile or as a `prebuild` step. This way, we don't have to maintain generated code, can reproduce this and stay up-to-date. Please feel freee to ask if you need help with this :)

Reply:
Hm, I agree with you. They were generated with [cxsd](https://www.npmjs.com/package/cxsd). The problem is, I might change a small number of definitions (currently working on that). E.g. the `_exists` or `_namespace` variables in `BaseType` are internal and should be optional. Another thing is, that there are fields of type `Date` or `number`, which is logically correct, but in the parsed JSON it is still a `string`. So it could be difficult to automate this completely... However, a manually run-command to update could be useful.

Reply:
Perhaps we could run `patch` on it after generating it if there is no other way?